.github/workflows/builder.yml

```diff
@@ -7,6 +7,12 @@ on:
     types: ["published"]
   push:
     branches: ["master"]
+    paths:
```

Comment:
You might want to trigger when this workflow changes as well?

Reply:
nope. That don't change the image